run-mypy.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+mypy --txt-report . w3lib tests
```

Comment:
Giving `tox -e typing` can be used instead, I think it may be better to remove this script.